client/src/use/useEditingLayer.js

```diff
@@ -0,0 +1,108 @@
+import { geojsonToBound, geojsonToBound2 } from '@/utils';
+
+export default function useEditingLayer({
+  // data
+  prompt, // TODO: refactor
+  frame,
+  detections,
+  tracks,
+  editingTrackId,
+  editingDetection,
+  // methods
+  setTrackEditMode,
+  deleteDetection,
+  setDetection,
+}) {
+  async function deleteTrack({ trackId }) {
+    const _detections = detections.value;
+    const result = await prompt({
+      title: 'Confirm',
+      text: `Please confirm to delete track ${trackId}`,
+      confirm: true,
```

Comment:
Adding a catch to either return or throw an error when an object is passed that doesn't contain trackId.  So if trackId is null.  Right now it will still ask for confirmation, iterate through the detections and then exit.